dts/bindings/shi/nuvoton,npcx-shi.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Serial Host Interface (SHI) node
+
+compatible: "nuvoton,npcx-shi"
+
+include: [base.yaml, pinctrl-device.yaml, shi-buffer-size.yaml]
```

Comment:
I was actually thinking more of doing `shi-device.yaml`, and then that could include `base.yaml`, and then this would only be `[shi-device.yaml, pinctrl-device.yaml]`. Does it make sense?

Reply:
Oh, sorry. Couldn't make free bindings due to the different method of CS handling, but cleaned definitely.

Reply:
Cool, one last bit: I think the pinctrl entries should stay on the higher level file, that's because the constraint change depending on the soc (such as having one or two entries), so by having them on the soc specific binding you can have correct "required" properties and assume they are going to be there on the driver.

Reply:
That sounds right, hope that now it is as you thought